toolkit/tools/pkg/imagecustomizerlib/baseconfigs_test.go

```diff
@@ -207,7 +207,11 @@ func TestBaseConfigsFullRun(t *testing.T) {
 	assert.Contains(t, string(moduleDisableContent), "vfio")
 
 	// Verify SELinux
-	verifyKernelCommandLine(t, imageConnection, []string{}, []string{"security=selinux", "selinux=1", "enforcing=1"})
+	hasUkis, err := baseImageHasUkis(imageConnection.Chroot())
```

Comment:
Don't dynamically check for stuff in tests. The test should know exactly what the OS image should contain.

Same comment below.

Reply:
Fixed.